pkgs/by-name/cb/cbconvert/gui.nix

```diff
@@ -0,0 +1,54 @@
+{
+  buildGoModule,
+  cbconvert,
+  gtk3,
+  lib,
+  stdenv,
+  wrapGAppsHook3,
+}:
+
+buildGoModule rec {
+  pname = "cbconvert-gui";
+
+  inherit (cbconvert)
+    patches
+    proxyVendor
+    src
+    tags
+    version
+    ;
+
+  nativeBuildInputs =
+    cbconvert.nativeBuildInputs
+    ++ lib.optionals (stdenv.hostPlatform.isUnix && !stdenv.hostPlatform.isDarwin) [
+      wrapGAppsHook3
+    ];
+  buildInputs =
+    cbconvert.buildInputs
+    ++ lib.optionals (stdenv.hostPlatform.isUnix && !stdenv.hostPlatform.isDarwin) [ gtk3 ];
```

Comment:
Since all other platforms are marked as broken and I suggested to remove them we might as well drop this condition.

Reply:
Done!